app/states/video_meeting_state.py

```
@@ -248,8 +263,46 @@ class VideoMeetingState(rx.State):
+        try:
+            client = await self._ensure_client()
+
+            # Load video meetings
+            raw_video_meetings = await client.fetch_video_meetings()
+            self.video_meetings = raw_video_meetings
+
+            # Load meeting recordings
+            # raw_recordings = await client.fetch_meeting_recordings()
+            # self.meeting_recordings = raw_recordings
+
+        except Exception as exc:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)